apps/nextra/pages/en/build/_meta.tsx

```diff
@@ -5,6 +5,7 @@ export default {
   sdks: "SDKs",
   indexer: "Indexer",
   cli: "CLI",
+  "create-aptos-dapp": "Create Aptos Dapp",
```

Comment:
I would keep it the same name as the tool

```suggestion
  "create-aptos-dapp": "create-aptos-dapp",
```